ledger-core-cosmos/src/cosmos/CosmosLikeAccount.cpp

```diff
@@ -109,10 +144,11 @@ namespace ledger {
                         out.trust = std::make_shared<TrustIndicator>();
                         auto fees = 0;
                         std::for_each(tx.fee.amount.begin(), tx.fee.amount.end(), [&] (cosmos::Coin amount) {
-                             fees += std::stoi(amount.amount);  // FIXME Is this correct??
+                                assert(amount.denom == "uatom"); // FIXME Temporary until all units correctly supported
+                                fees += BigInt::fromDecimal(amount.amount).toInt();
                         });
                         out.fees = BigInt(fees);
-                        out.type = api::OperationType::SEND; // FIXME Manage operation type correctly
+                        out.type = api::OperationType::NONE; // FIXME Figure out if this is enough for Cosmos
```

Comment:
Almost certainly not, but we will add a dispatch to handle a `api::CosmosLikeMsgType <-> api::OperationType` mapping. Either as a static function in `CosmosLikeMsgType` or as free function in `CosmosLikeConstants.hpp` (I like the 2nd one better for now)